src/Backends/PrefersColorSchemeSettings.vala

```diff
@@ -146,15 +146,14 @@ public class SettingsDaemon.Backends.PrefersColorSchemeSettings : Object {
         AM–AM (time_double > from && time_double < to)
         PM–PM (^ same as above)
         AM–PM (^ same as above)
-        PM–AM (time_double > from && time_double < to + 24) so only one odd case.
+        PM–AM (time_double > from && time_double < (to + 24)) so only one odd case.
         */
 
         if (to < from) {
```

Comment:
As long as we're here and switching things around… is there a reason not to use `if (from > to) {` here? It seems to me like that makes more sense and is what you're actually checking for, given the order of the items in the UI: if `from` is _after_ `to`, and thus it's AM to PM…

Reply:
> is there a reason not to use `if (from > to) {` here?

If you mean since the comment shows PM-AM so the code should follow suite then yeah it makes sense to put `if (from > to). Applied your suggestion, thanks!